dev/TabView/TabView.xaml

```diff
@@ -606,33 +600,16 @@
                                     </VisualState.Setters>
                                 </VisualState>
                             </VisualStateGroup>
+                            <VisualStateGroup x:Name="TabSeparatorVisbilityStates">
+                                <VisualState x:Name="TabSeparatorVisible"/>
+                                <VisualState x:Name="TabSeparatorHidden">
+                                    <VisualState.Setters>
+                                        <Setter Target="TabSeparator.Opacity" Value="0"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
                         </VisualStateManager.VisualStateGroups>
 
-                        <Path 
-                            x:Name="LeftRadiusRender"
-                            x:Load="False"
-                            Grid.Column="0"
-                            Visibility="Collapsed"
-                            VerticalAlignment="Bottom"
-                            Height="{Binding Source={ThemeResource OverlayCornerRadius}, Path=BottomLeft}"
-                            Margin="{Binding Source={ThemeResource OverlayCornerRadius},
-                                Converter={StaticResource TabViewLeftInsetCornerConverter}}"
-                            Stretch="Uniform"
-                            Fill="{ThemeResource TabViewItemHeaderBackgroundSelected}"
-                            Data="M4 0 L4 4 L0 4 A4,4 90 0 0 4 0 Z" /> 
```

Comment:
What issues exactly would this create? Still trying to wrap my head around the removal of this since it's one of the things making this control more fluent (looking at edge for example).

Reply:
First of all, it bring lots of code changes to make it properly work because now we have strokes everywhere and to make adjacent tabs (to the selected tab) smoothly connect with the arcs we need to give them corner radius only on one side.
    Second problem, this arc require negative margin from both sides to bring adjacent tabs right next to the selected one - so when you have selected tab as the very first or very last tab - it brings the footer and header closer meaning there's different spacing between unselected and selected tab when they are being 1st/last tab. It makes it almost impossible to connect header/footer with the tab in a continuous line :)

Third problem, the background of the arc as well as the selected tab itself is semi-transparent what makes anything behind it affect it. So when we have unselected tab hover background applied next to the selected one - it will be visible on the arc because it's technically "behind" the arc :)

Reply:
> First of all, it bring lots of code changes to make it properly work because now we have strokes everywhere and to make adjacent tabs (to the selected tab) smoothly connect with the arcs we need to give them corner radius only on one side.

Don't think you would need to do that, after all we didn't have to do it now.

> Second problem, these arc require negative margin from both sides to bring adjacent tabs right next to the selected one - so when you have selected tab as the very first or very last tab - it brings the footer and header closer meaning there's different spacing between unselected and selected tab when they are being 1st/last tab. It makes it almost impossible to connect header/footer with the tab in a continuous line :)

Could add a know or margin for this I guess, but wouldn't be much fun.

> Third problem, the background of the arc as well as the selected tab itself is semi-transparent what makes anything behind it affect it. So when we have unselected tab hover background applied next to the selected one - it will be visible on the arc because it's technically "behind" the arc :)

Oh so the tabs are semi transparent now? Didn't know that, that makes sense then.

Guess they are gone for good (or for worse) then...

Reply:
yup, because of semi transparency we have to do face lots of issues :)
it's probably possible to make it work, but it will require tons of time to figure out exactly how :)

Reply:
@stmoy @StephenLPeters Is this worth a tracking issue to reintroduce?

Reply:
@chingucoding Yes! I do agree that we want the bottom arcs. We just haven't figured out a good solution due to the transparency and this is a less than ideal compromise.